yasunori.toml

```
@@ -1,3 +1,318 @@
+"""
+
+[[yasunori]]
+id = 89
+title = "yasunori house と呼ばれているの本当によくわからない"
+date = "2024-11-21"
+at = "vim-jp #times-ryoppippi"
+senpan = "thinca"
+content = """
+手配したの ryoppippi さんのはずなのに yasunori house と呼ばれているの本当によくわからない
+"""
+meta = """
+- memo
+  - ryoppippiさんがVimConfの前後で泊まるために手配した宿がなぜかyasunori houseと呼ばれていた
+    - ちなみに結果的にyasunori(原義)を含むyasunori達が泊まっておりyasuno率が非常に高かったが、手配時点で全くyasunori関係なかった
+  - yasunori0418「yasunoriってなんなんだよ」
+"""
+
+[[yasunori]]
+id = 88
+title = "いよいよ明日がyasunori meetup本番ですよ！"
+date = "2024-11-20"
+at = "vim-jp #times-yasunori"
+senpan = "tomoya"
+content = """
+いよいよ明日がyasunori meetup本番ですよ！
+むっちゃドキドキしてきた…。
+yasunoriの皆さん、今日くらいはddは休んで明日に備えますよね？
+"""
+meta = """
+- memo
+  - https://dic.nicovideo.jp/a/%E3%81%84%E3%82%88%E3%81%84%E3%82%88%E6%98%8E%E6%97%A5%E3%81%8C%E3%82%BB%E3%83%B3%E3%82%BF%E3%83%BC%E8%A9%A6%E9%A8%93%E6%9C%AC%E7%95%AA%E3%81%A7%E3%81%99%E3%82%88%21
+    - もちろんyasunori meetup #3は今日行われた
+  - 「yasunori meetup明日か！！今日はゆっくり寝よう」
+"""
+
+[[yasunori]]
+id = 87
+title = "yasunori を dd すれば yasunori だ"
+date = "2024-11-19"
+at = "vim-jp #times-yasunori"
+senpan = "NI57721"
+content = """
+yasunori を dd すれば yasunori だ
+"""
+meta = """
+- memo
+  - yasunoriという名を冠するデバイスをddで消去する(yasunoriする)構図
+  - `# dd if=/dev/zero of=/dev/yasunori bs=4M status=progress`
+"""
+
+[[yasunori]]
+id = 86
+title = "yasunoriがyasunoriだけじゃなくなってyasunoriになった"
+date = "2024-11-16"
+at = "vim-jp #times-yasunori"
+senpan = "takeokunn"
+content = """
+お子さんもyasunoriがyasunoriだけじゃなくなってyasunoriになったのビックリするだろうな
+"""
+meta = """
+- memo
+  - ryoppippiがtomoyaハウスに遊びに行ってお子さんと遊んでいる。その時点ではまだryoppippiはyasunoriではなかった
+  - お子さんはyasunori(原義)も見たことがあるらしい
+  - その後、ryoppippiは暴食のyasunoriになった
+  - 「なんだろう、いままでは分からんで押し通していたのに、ryoppippiもyasunoriっていうのすぐに理解しちゃった」
+"""
+
+[[yasunori]]
+id = 85
+title = "yasunoriが22人いればドリームチーム"
+date = "2024-11-15"
+at = "vim-jp #times-yasunori"
+senpan = "nekowasabi"
+content = """
+:yasunori0418: が22人いればドリームチーム
+"""
+meta = """
+- memo
+  - 元ネタはアイシールド21の金剛阿含
+  - :yasunori0418: はvim-jp Slackでyasunori(原義)氏のアイコンとして表示される
+"""
+
+[[yasunori]]
+id = 84
+title = "まーでも俺yasunoriだしなー"
+date = "2024-11-14"
+at = "vim-jp #times-yasunori"
+senpan = "takeokunn"
+content = """
+まーでも俺yasunoriだしなー
+"""
+meta = """
+- memo
+  - #times-yasunoriが#times-take化していることに対する本人のコメント
+  - 「yasunori(原義)とは「別にtimes適当に使っていいよ」という話を合意取ってるのでその辺は大丈夫(一応)」らしい
+"""
+
+[[yasunori]]
+id = 83
+title = "yasunoriも知らないyasunoriの世界"
+date = "2024-11-05"
+at = "vim-jp #times-yasunori"
+senpan = "takeokunn"
+content = """
+yasunoriも知らないyasunoriの世界
+"""
+meta = """
+- memo
+  - 本人も「yasunoriってなんなんだろうな」と言っており、最早誰にも分からないミームと化している
+"""
+
+[[yasunori]]
+id = 82
+title = "合意の上のyasunori"
+date = "2024-11-03"
+at = "vim-jp #times-yasunori"
+senpan = "kuuote"
+content = """
+合意の上のyasunori
+"""
+meta = """
+- memo
+  - USBポートを開けたまま放置されたPCを見てもUSBメモリを差し込んではいけない
+  - yasunori「まぁ、同意が取れれば消してパーティションを切るところまでやりますが」
+    - ↑実際にたけてぃのthinkpadは依頼があってやりましたよ
+  - とあるように、同意が取れていることが大事
+"""
+
+[[yasunori]]
+id = 81
```

```suggestion
id = 83
```

とりあえずコンフリクト解決しつつ、idずらしますか